apps/condo/domains/meter/constants/registerMetersReadingsService.js

```diff
@@ -0,0 +1,90 @@
+const { GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+
+const {
+    TOO_MUCH_READINGS,
+    ORGANIZATION_NOT_FOUND,
+    PROPERTY_NOT_FOUND,
+    INVALID_METER_VALUES,
+    MULTIPLE_METERS_FOUND,
+    INVALID_ACCOUNT_NUMBER,
+    INVALID_METER_NUMBER,
+    INVALID_DATE,
+} = require('@condo/domains/meter/constants/errors')
+
+
+const ISO_DATE_FORMAT = 'YYYY-MM-DD'
+const EUROPEAN_DATE_FORMAT = 'DD.MM.YYYY'
+const DATE_PARSING_FORMATS = [
+    'YYYY-MM-DDTHH:mm:ss.SSS[Z]', // The result of dayjs().toISOString()
+    `${ISO_DATE_FORMAT} HH:mm:ss`, `${EUROPEAN_DATE_FORMAT} HH:mm:ss`, // Up to seconds
+    `${ISO_DATE_FORMAT} HH:mm`, `${EUROPEAN_DATE_FORMAT} HH:mm`, // Up to minutes
+    ISO_DATE_FORMAT, EUROPEAN_DATE_FORMAT, // No time
+    'YYYY-MM', 'MM-YYYY', 'YYYY.MM', 'MM.YYYY', // Some exotic cases
+]
+const READINGS_LIMIT = 500
+
+const ERRORS = {
+    TOO_MUCH_READINGS: (sentCount) => ({
```

Comment:
Please, don't use functions. We already have interpolation and you don't need to use functions